.golangci.yaml

```diff
@@ -28,10 +27,19 @@ linters:
     - megacheck
     - misspell
     - revive
-    - varcheck
+    - unused
     # TODO(gbelvin): write license linter and commit to upstream.
     # ./scripts/check_license.sh is run by ./scripts/presubmit.sh
 
 issues:
   # Don't turn off any checks by default. We can do this explicitly if needed.
   exclude-use-default: false
+  # List of regexps of issue texts to exclude.
```

Comment:
Out of interest, how many packages are there which need docstrings?
We could add them and then reenable this...

Reply:
```
➜  trillian-examples git:(master) ✗ golangci-lint run --print-issued-lines=false --max-same-issues=0 | wc -l
12
```

```
➜  trillian-examples git:(master) ✗ golangci-lint run --print-issued-lines=false --max-same-issues=0
binary_transparency/firmware/api/annotation.go:15:1: package-comments: should have a package comment (revive)
binary_transparency/firmware/internal/client/client.go:15:1: package-comments: should have a package comment (revive)
binary_transparency/firmware/cmd/flash_tool/devices/device.go:15:1: package-comments: should have a package comment (revive)
witness/golang/omniwitness/usbarmory/internal/storage/persistence.go:15:1: package-comments: should have a package comment (revive)
sumdbaudit/cli/clone/clone.go:15:1: package-comments: should have a package comment (revive)
binary_transparency/firmware/devices/dummy/common/measurement.go:15:1: package-comments: should have a package comment (revive)
binary_transparency/firmware/devices/usbarmory/measurement.go:15:1: package-comments: should have a package comment (revive)
experimental/batchmap/sumdb/mapdb/tiledb.go:15:1: package-comments: should have a package comment (revive)
witness/golang/omniwitness/configs.go:15:1: package-comments: should have a package comment (revive)
sumdbaudit/cli/mirror/mirror.go:15:1: package-comments: should have a package comment (revive)
witness/golang/internal/persistence/testonly/persistence.go:15:1: package-comments: should have a package comment (revive)
binary_transparency/firmware/devices/dummy/rom/rom.go:15:1: package-comments: should have a package comment (revive)
```